src/lang-promql/complete/index.ts

```diff
@@ -37,6 +37,12 @@ export interface CompleteConfiguration {
   lookbackInterval?: number;
   httpErrorHandler?: (error: any) => void;
   fetchFn?: FetchFn;
+  // cache will allow user to change the configuration of the cached Prometheus client (which is used by default)
+  cache?: {
+    // maxAge is the maximum amount of time that the cache will keep in memory the data fetched from Prometheus.
```

Comment:
This is not quite true, as elements > `maxAge` are still kept in memory until you try to retrieve them again. The purpose of `maxAge` seems more to remove outdated / stale elements from the cache so that they are fetched again with an updated version. So a better description for the library user could be:

```suggestion
    // maxAge is the maximum amount of time that a cached completion item is valid before it needs to be refreshed.
```